application/feature/characters/favorite/api/src/main/java/com/stepango/blockme/feature/characters/favorite/api/domain/usecase/IGetCharacterFavoriteUseCase.kt

```diff
@@ -1,8 +1,8 @@
 package com.stepango.blockme.feature.characters.favorite.api.domain.usecase
 
-import com.stepango.blockme.feature.characters.favorite.api.domain.model.ICharacterFavorite
+import com.stepango.blockme.feature.characters.core.api.domain.model.ICharacter
 
 interface IGetCharacterFavoriteUseCase {
 
-    suspend operator fun invoke(characterFavoriteId: Long): ICharacterFavorite?
+    suspend operator fun invoke(characterFavoriteId: Long): ICharacter?
```

Comment:
I think usecase shouldn't throw nullable behavior for "clients", but for improving that need another issue)